src/server.js

```diff
@@ -66,7 +66,7 @@ const init = async () => {
 		log(`User sent account creation request for account '${username}'`)
 
 		if (!validUsername(username)) {
-			const message = 'User account username was not valid'
+			const message = 'User account username was not valid. It can only contain alphanumerical or underscore characters. It\'s length must be between 2 and 20.'
```

Comment:
Can you change the text to say this instead?

Account username can only contain alphanumerical and underscore characters and its length must be between 2 and 20.